userguide/system.rst

```diff
@@ -2333,11 +2333,14 @@ complete these fields:
   issue that describes the problem, and if applicable, what steps can
   be taken to reproduce it.
 
-After completing the fields, click the :guilabel:`SUBMIT` button to
-automatically generate and upload the report to
-https://redmine.ixsystems.com/projects/freenas/issues. A pop-up menu
-provides a URL so to view status or add additional information to the
-report.
+Click :guilabel:`SUBMIT` to automatically generate and upload the report
+to the
+`bug tracker <https://redmine.ixsystems.com/projects/freenas/issues>`__.
+This process can take several minutes while information is collected and
+sent.
+
+After the new ticket is created, the URL is shown for updating with more
```

Comment:
The other branch version was "viewing or updating".

Reply:
It was, but you suggested changing it to the way it appears now.

Reply:
https://github.com/freenas/freenas-docs/pull/625#discussion_r250245464

Reply:
Sorry, my mistake.  I have fixed it.